### PR TITLE
fix the issue of cdn command -option not recongizable issue

### PR DESCRIPTION
**Pull request by csmengwan**
The content to be added to Changelog is as follows:
* improved help for cdn commands
* 
*

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@csmengwan__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (qinwa). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: lib/commands/arm/cdnmanagement/cdnmanagement._js [hunk 1]
```diff
@@ -89,11 +89,11 @@ exports.init = function(cli) {
     });
 
   // Get profile
-  profiles.command('show <name> <resource-group>')
+  profiles.command('show [name] [resource-group]')
     .description($('Show the infomation of a specific cdn profile'))
     .usage('[options] <name> <resource-group>')
-    .option('-n, --name [name]', $('Name of the Cdn Profile'))
-    .option('-g, --resource-group [resource-group]', $('Name of the Resource Group'))
+    .option('-n, --name <name>', $('Name of the Cdn Profile'))
```
Comment (haocs) [inline lib/commands/arm/cdnmanagement/cdnmanagement._js]: @csmengwan - I think you need to add some validation logic to make sure those required arguments are provided. Please take a look at the following links: https://github.com/Azure/azure-xplat-cli/blob/dev/lib/commands/arm/vm/vm._js#L209 https://github.com/Azure/azure-xplat-cli/blob/dev/lib/commands/arm/webapp/webapp._js#L98

**Comment by haocs** [comment]
@csmengwan - could you please address the comment? If the required is omitted, it ends up with a very confusing error message. It will difficult for users to figure out what went wrong.

File: Deploy14800.json [hunk 2]
```diff
@@ -0,0 +1,300 @@
+{
```
Comment (amarzavery) [inline Deploy14800.json]: @csmengwan - Why have you checked this file? It seems its a test file. Is this a by product of one of your tests?

Comment (amarzavery) [inline Deploy14800.json]: If yes, then please try to create this file inside test/output directory. That directory is already gitignored.

**Pull request merged**